src/components/Contacts/ContactDetails/ContactDetailsTab/People/ContactDetailsTabPeople.tsx

```diff
@@ -0,0 +1,141 @@
+import { Avatar, Box, styled, Typography } from '@material-ui/core';
+import { Cake, Email, Phone } from '@material-ui/icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Person } from '../../../../../../graphql/types.generated';
```

Comment:
It's better to use the query's types instead of these global types since they may have fields that weren't actually queried.

You could probably make another fragment on `Person` that could be used in `Contact.primaryPerson` and `Contact.people.nodes` which would give you a nice type to use here.